### ISSUE TITLE
Blocking www.flyertown.ca prevents eflyers from being displayed

### ISSUE THREAD

**Issue by ck9ine** [lightswitch05/hosts#224]
londondrugs.com uses "eflyer.londondrugs.com" to serve up their store flyers.

eflyer.londondrugs.com CNAME www.flyertown.ca

Would you consider removing flyertown.ca and www.flyertown.ca please?  The other flyertown.ca subdomains on the blocklist do not seem to interfere with this, and thus they can remain.

**Comment by lightswitch05** [comment]
Added in #68931e40245bca3d62d376629cbb203d212a530a

Added in https://github.com/lightswitch05/hosts/commit/68931e40245bca3d62d376629cbb203d212a530a with a useless commit message


Looking at it, I see it's just a CNAME for `circularhub.com` - which I've purposefully not added to my list. A quick look at other lists shows in the only one blocking it. Since I don't have any good records of why I added this- combined with it breaking things and me being the only one blocking it- I think I'll just remove the entire domain from the list. If I ever see one of the domains doing tracking then I'll add it back.
